WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift

```diff
@@ -690,24 +690,24 @@ extension GutenbergViewController: GutenbergBridgeDelegate {
             self.showAlertForReplacingFeaturedImage(mediaID: mediaID)
         }
 
-        if presentedViewController != nil {
+        if let viewController = presentedViewController, !viewController.isBeingDismissed {
```

Comment:
Now that we trigger this alert when the image is replaced, the previous code erroneously dismissed the entire editor itself. This occurred as the Media Picker modal was already actively `isBeingDismissed` when `dismiss` was invoked. By checking `isBeingDismissed`, we essentially opt to not dismiss the `presentedViewController` if it is already on its way out.

Reply:
Nice, thank you for expanding on that!